crates/marine-test-macro-impl/src/marine_test/token_stream_generator/service_generator.rs

```diff
@@ -0,0 +1,282 @@
+/*
+ * Copyright 2021 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use crate::attributes::{ServiceDescription};
+use crate::TResult;
+use crate::TestGeneratorError;
+use crate::marine_test::config_utils::{Module, ProcessedService};
+use crate::marine_test::utils::new_ident;
+use crate::marine_test::{modules_linker, config_utils};
+
+use proc_macro2::TokenStream;
+use quote::quote;
+
+use std::path::Path;
+use std::path::PathBuf;
+use itertools::{Itertools, zip};
+use std::collections::HashMap;
+use crate::marine_test::modules_linker::{LinkedModules, LinkedModule, UseDescription};
```

Comment:
Could you please rearrange imports in such a way that imports from this project crate will come first, then from other crates, and finally from std.

Reply:
Sure, will fix it. Btw, can we force `rustfmt` to reorder imports in the right way?

Reply:
rustfmt has an option for reordering imports but afaik it can't do that in the necessary way.